README.md

```diff
@@ -2,7 +2,7 @@
 
 **[ffmpeg](https://ffmpeg.org) static binaries for Mac OSX, Linux, Windows and FreeBSD.**
 
-Supports macOS (64-bit), Linux (32 and 64-bit, armhf, arm64), Windows (32 and 64-bit) and FreeBSD (64-bit). [The ffmpeg version currently used is `4.3.2`.](https://github.com/eugeneware/ffmpeg-static/releases/tag/b4.3.2)
+Supports macOS (64-bit and arm64), Linux (32 and 64-bit, armhf, arm64), Windows (32 and 64-bit) and FreeBSD (64-bit). [The ffmpeg version currently used is `4.3.2`.](https://github.com/eugeneware/ffmpeg-static/releases/tag/b4.3.2)
```

Comment:
@derhuerst should we also bump the versions of the other builds since the Apple Silicon build will be v4.4?

Reply:
I will do the rest once all binary sources mentioned in https://github.com/eugeneware/ffmpeg-static/issues/78 provide 4.4 (just FreeBSD missing by now I think).